src/main/sass/main.scss

```diff
@@ -169,3 +169,8 @@ table {
   }
 }
 
+//link
+
+.no-underline a {
+    text-decoration: none;
```

Comment:
why did you add this class? Can we have a consistent approach to link styling? eg I've started adding `a:hover { text-decoration: underline; }` but this link will not have this behaviour.